src/oge/validation.py

```diff
@@ -805,9 +799,6 @@ def test_emissions_adjustments(df, year):
                     [
                         "report_date",
                         "plant_id_eia",
-                        "generator_id",
-                        "prime_mover_code",
-                        "energy_source_code",
```

Comment:
Instead of removing these entirely, could we just implement a flexible column handling like:
```
columns_to_show = ["report_date",
                        "plant_id_eia",
                        "generator_id",
                         "emissions_unit_id_epa",
                        "prime_mover_code",
                        "energy_source_code",]
...
bad_adjustment[[col for col in columns_to_show if col in bad_adjustment.columns] + [f"{pollutant}_mass_lb",f"{pollutant}_mass_lb_adjusted",]]
```

Reply:
I will do that

Reply:
Done